<?xml version="1.0"?>
<launch>
    <node pkg="fbot_llm_agent" type="crewai_agent_node.py" name="crewai_agent_node">
        <rosparam file="$(find fbot_llm_agent)/config/crewai_agent.yaml" command="load" />
    </node>
</launch>